src/rules/declaration-block-no-duplicate-properties/index.js

```diff
@@ -62,6 +64,10 @@ export default function (on, options) {
             return
           }
 
+          const ignoreProperties = options && options.ignoreProperties || []
+
+          if (ignoreProperties.indexOf(prop.toLowerCase()) !== -1) { return }
```

Comment:
This could use the `matchesStringOrRegexp()` util to allow users to insert regular expressions as well as strings, e.g. http://stylelint.io/user-guide/rules/function-name-case/#optional-options